integration-tests/byoid/integration_test.go

```diff
@@ -37,9 +37,14 @@ import (
 	"log"
 	"net/http"
 	"net/http/httptest"
+	neturl "net/url"
```

Comment:
nit: Don't rename this import. Rename a variable if required.

Reply:
As a head's up, this rename is actually copied from the original implementation in `oauth2`.  Should I rename it there as well?

Reply:
We can fix it next time there are some changes to be made over there.